pages/common/elm.md

```diff
@@ -0,0 +1,15 @@
+# elm
+
+> Run and manage programs in Elm programming language.
+
+- Run interactive Elm shell (repl):
```

Comment:
"Start an" instead of "Run" sounds better. Although, looking at the `node`, `python` and `ruby` pages, all have slightly different styles.

Reply:
Ok, I will change it